### PR TITLE
Add support for a few common LLMs in the text benchmark script

### PR DESCRIPTION
**Pull request by vid-koci**
Adds support for RoBERTa, Electra, and Deberta finetuning. Picks the batch size depending on the model.

### CODE REVIEW

File: benchmark/data_frame_text_benchmark.py [hunk 1]
```diff
@@ -73,6 +75,9 @@
     default="sentence-transformers/all-distilroberta-v1",
     choices=[
         "distilbert-base-uncased",
+        "roberta-large",
+        "microsoft/deberta-v3-large",
```
Comment (zechengz) [inline benchmark/data_frame_text_benchmark.py:79]: For now maybe we can just follow the paper https://arxiv.org/pdf/2111.02705.pdf just include roberta and electra

File: benchmark/data_frame_text_benchmark.py [hunk 2]
```diff
@@ -131,11 +153,10 @@ def __init__(self, model: str):
         elif model == "sentence-transformers/all-distilroberta-v1":
             target_modules = ["intermediate.dense"]
         else:
-            raise ValueError(f"Model {model} is not specified for "
-                             f"LoRA finetuning.")
+            target_modules = "all-linear"
```
Comment (zechengz) [inline benchmark/data_frame_text_benchmark.py:156]: Do we have experiments on finetuning which (sub)layers can achieve better performance?

**Review by zechengz — APPROVED** [review]
In general LGTM, left some comments

**Review by weihua916 — APPROVED** [review]
Thanks!

File: benchmark/data_frame_text_benchmark.py (hunk 2 above)
Comment (vid-koci) [reply benchmark/data_frame_text_benchmark.py:156]: Not yet, hence the default.

File: benchmark/data_frame_text_benchmark.py (hunk 1 above)
Comment (vid-koci) [reply benchmark/data_frame_text_benchmark.py:79]: I've added DeBERTa, but you're right, I don't think we'll be running experiments with it unless necessary or unless we have plently of extra time, which usually doesn't happen.

**Pull request merged**